tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -189,6 +191,52 @@ def should_exclude_files_type(file_key: str, exclude_file_types: List[str]) -> b
     return False
 
 
+def set_optional_event_property(event: Dict[str, Any], target_path_cursor: Dict,
```

Comment:
looks like we updated the `request_from_archive` code last time but forgot to update this lambda as well.
Good catch.